tests/integ/modin/series/test_compare.py

```diff
@@ -10,8 +10,8 @@
 from tests.integ.modin.sql_counter import sql_count_checker
 from tests.integ.modin.utils import create_test_series, eval_snowpark_pandas_result
 
-# (+1 query, +0 join) materialize first series's index for comparison
-# (+1 query, +0 join) materialize second series's index for comparison
+# (+1 query, +0 join) materialize first series's index for comparison if multi-index
```

Comment:
Why "if multi-index"? Thequery count seems to be constant.

Reply:
Our lazy index implementation does not support multi-index yet. https://github.com/snowflakedb/snowpark-python/blob/main/src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py#L1581

Reply:
I knew that, but I was confused because I didn't notice the query count for non-multiindex. We have `query_count=4, join_count=4` for the default index, so LGTM

Reply:
I see